cumulus/steps/ec2/dns.py

```diff
@@ -0,0 +1,53 @@
+from troposphere import route53
+from troposphere import (
+    Ref, Join, ec2)
+from cumulus.chain import step
+
+
+class Dns(step.Step):
+
+    def __init__(self,
+                 base_domain,
+                 hosted_zone_id,
+                 dns_name,
+                 ):
+
+        step.Step.__init__(self)
+
+        self.base_domain = base_domain
+        self.hosted_zone_id = hosted_zone_id
+        self.dns_name = dns_name
+
+    def handle(self, chain_context):
+        template = chain_context.template
+
+        # ALB_RECORDSET_NAME = "AlbAlias"
+        # name = Join('', [ALB_RECORDSET_NAME, Ref("stackname")]),
+
+        name = 'AlbAlias'
+
+        template.add_resource(route53.RecordSetGroup(
+            "Route53Records",
+            RecordSets=[
+                route53.RecordSet(
+                    name,
+                    Weight=1,
+                    SetIdentifier="original",
+                    AliasTarget=route53.AliasTarget(
+                        HostedZoneId=self.hosted_zone_id,
+                        DNSName=self.dns_name,
+                        EvaluateTargetHealth=False,
+                    ),
+                    Name=Join("", [
+                        Ref("stackname"),
```

Comment:
Not sure if this was brought over from the builder project but we have a concept of self.name,   chain_context.instance_name,  and now Ref("stackname").  

Can we streamline these ? Is stackname really required?

Reply:
Probably not required, can use the other options if they are providing a stackname.